docs/runbook-trec2018-anserini.md

```diff
@@ -83,28 +83,19 @@ Metric            | AP*    | AP+    | NDCG*  | NDCG+  | P10*   | P10+   |
 The same as Core Track
 
 ### Submitted Runs
-```
```

Comment:
Why remove the commands to generate the submitted runs? 
I think it is extremely important for reproducing.

Reply:
I thought lines 87 - 91 were essentially the same as 100 - 104.

Reply:
I see... thanks!